scripts/k6/stream-api/static.js

```diff
@@ -0,0 +1,46 @@
+import * as common from "./common.js";
+import { checkResponse, post } from "./common.js";
+
+export const options = common.options;
+
+export function setup() {
+  let source = common.setupSource();
+
+  let sink = common.setupSink(
+    source.id,
+    {
+      sinkId: "test-sink",
+      name: "test-sink",
+      type: "table",
+      table: {
+        type: "keboola",
+        tableId: "in.c-bucket.test-sink",
+        mapping: {
+          columns: [
+            { type: "uuid-v7", name: "id", primaryKey: true },
+            { type: "datetime", name: "datetime" },
+            { type: "ip", name: "ip" },
+            { type: "body", name: "body" },
+            { type: "headers", name: "headers" },
+          ],
+        },
+      },
+    },
+  )
+
+  const payload = { a: "b", c: { d: "e", f: { g: "h" } } };
+  const headers = {
+    "My-Custom-Header": "custom header value abcd",
+  };
+
+  return { source, sink, payload, headers };
+}
+
+export function teardown(data) {
+  common.teardownSource(data.source.id)
```

Comment:
Missing teardown of sink. I checked that it is possible to leave teardown of source it disallows the sink. Do we want to leave it like that?